YCAI/src/components/dashboard/UserProfileBox.tsx

```diff
@@ -51,13 +75,40 @@ export const LoggedUserProfileBox: React.FC<LoggedUserProfileBoxProps> = ({
       </Box>
       </Box>
       <Button
-          variant="contained"
-          size="small"
-          onClick={() => onLogout()}
-          style={{ marginLeft: 50 }}
+        variant="contained"
+        size="small"
+        onClick={handleUnlinkClicked}
+        style={{ marginLeft: 50 }}
+      >
+        {t('actions:unlink_channel')}
+      </Button>
+      {unlinkClicked && (
```

Comment:
Can you move this component out and link it also to the "unlink" button in settings page?

Reply:
yup